src/components/quickStarts.stories.mdx

```diff
@@ -50,48 +50,64 @@ const MyApp = () => {
 };
 ```
 
-> **Note:** Joining a meeting in view-only mode may result in attendees not being able to join the meeting successfully. This results in an issue where the roster remains empty and the attendee cannot receive audio and video feeds from others. Check the [GitHub Issue](https://github.com/aws/amazon-chime-sdk-js/issues/474) for details.
+> **Note:** Joining a meeting in view-only mode may result in attendees not being able to join the meeting successfully. This results in an issue where the roster remains empty and the attendee cannot receive audio and video feeds from others. Check the <a target="_blank" href="https://github.com/aws/amazon-chime-sdk-js/issues/474">GitHub Issue</a> for details.
 >
 > The Amazon Chime SDK for JavaScript uses the attendee's audio signal to determine whether or not this attendee has joined a meeting. However, the browser's autoplay policy sometimes prohibits this audio signal from being played automatically. You could fix this issue by enabling the autoplay policy:
 >
 >* Chrome: Preference - Privacy and security - Find your project hosting URL - Sound - Allow
 >* Safari: Preference - Auto-Play - Find your project hosting URL - Allow All Auto-Play
 >* Edge: Preference - Cookies and Site Permissions - All permissions - Media autoplay - Add your project hosting URL
 
-## Handle Device Permission
+## Handling How to Get Device Permissions From the Browser
 
-By default, after `join()` method is called, `MeetingManager` will:
+Amazon Chime React Component Library use [`DeviceLabelTrigger`](https://github.com/aws/amazon-chime-sdk-js/blob/bec0e13331ee1ff9198070dd2a6bb1bd289dd14e/src/devicecontroller/DeviceControllerFacade.ts#L25-L37) to call `getUserMedia` API to invoke the device permission prompt to request device permission from browser. `MeetingManager.join()` API provides an optional parameter `options: MeetingManagerJoinOptions` which contains property `deviceLabels: DeviceLabels | DeviceLabelTrigger` to configure `DeviceLabelTrigger`.
```

Comment:
Can simplify the sentence by dropping "to call `getUserMedia` API to invoke the device permission prompt" from this sentence as it is an internal detail and builders may not understand it.

Reply:
I add this sentence to explicitly call out the essence of `DeviceLabelTrigger`. This is useful to reduce confusion so prefer keeping it.